src/playground/components/Alert.js

```diff
@@ -70,12 +70,26 @@ export default function Alert({ type, text, message, onFix }) {
                     </ul>
                 </>
             ) : (
-                fix && (
-                    <button onClick={() => onFix(message)} className="alert__fix-btn">
+                <div style={{
+                    display: "flex"
+                }}>
+                    { fix && (
+                        <button onClick={() => onFix(message)} className="alert__fix-btn" style={{ "border-radius": 0 }}>
```

Comment:
```suggestion
                        <button onClick={() => onFix(message)} className="alert__fix-btn">
```
Pls, avoid inline style. Try to add style to the existing class.